pkg/operator/ceph/cluster/cluster.go

```diff
@@ -65,26 +65,7 @@ func newCluster(c *cephv1.CephCluster, context *clusterd.Context) *cluster {
 		ownerRef: ClusterOwnerRef(c.Namespace, string(c.UID))}
 }
 
-func (c *cluster) setCephMajorVersion(timeout time.Duration) error {
```

Comment:
This function was an attempt to "set" the version for the orchestration. It was cleaner to get rid of this helper and just let the caller use the version in the orchestration directly.